fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/impl/BIOAsyncMonix.scala

```diff
@@ -0,0 +1,132 @@
+package izumi.functional.bio.impl
+
+import java.util.concurrent.CompletionStage
+
+import izumi.functional.bio.BIOExit.MonixExit._
+import izumi.functional.bio.{BIOAsync, BIOExit, BIOFiber}
+import monix.bio
+import monix.bio.{Cause, IO, Task}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+object BIOAsyncMonix extends BIOAsyncMonix
+
+class BIOAsyncMonix extends BIOAsync[IO] {
```

Comment:
Do you think `BIOTemporal` is implementable for monix-bio? (or monix-bio + `cats.effect.Clock`?)

Reply:
I don't thinks so. I haven't spotted any functionality to repeat/retry using Schedulers. Um, maybe because `cats-retry` exists, so they though the one could use it with monix bio.

Reply:
@VladPodilnyk `BIOTemporal#retryOrElse` does not require Schedule though, it's easily implementable manually (you should decrease the remaning duration after each failed try and execute `orElse` once the alloted time for retrying is up)

Reply:
Um, do u mean decrease by effect execution time? Otherwise, it's just does not make any sense, 'cuz it won't be related to a time, but to number of retries, am I right?

Reply:
@VladPodilnyk Yeah